CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Config: Increased default amount for minimal channel capacity from 1k sat to 10k sat.
 - Build: Non-developer builds are now done with "-Og" optimization.
 - JSON API: `pay` will no longer return failure until it is no longer retrying; previously it could "timeout" but still make the payment.
+- JSON API: `fundchannel` now accepts an utxo array to use to fund the channel as the second parameter (in addition to 'all' or an amount in sats)
```

Comment:
nit: instead of saying `in addition to`, I'd reword this to `instead of` as you can only provide one of the three ('all', satoshis, or a utxo set)

Reply:
Ok. I appreciate that you review my english too, since I'm not a native speaker.